platform/android/java/app/config.gradle

```diff
@@ -1,5 +1,5 @@
 ext.versions = [
-    androidGradlePlugin: '3.5.3',
+    androidGradlePlugin: '4.0.0',
```

Comment:
This change should be a separate PR that fixes any issues with the migration to 4.0.0 (e.g. specified path to ndk)